marlowe-playground-client/src/Simulation.purs

```diff
@@ -623,11 +648,11 @@ inputItem state isEnabled person (MoveToSlot slot) =
     else
       []
 
-  inFuture = view (_currentMarloweState <<< _slot) state < slot
+  inFuture = maybe false (\x -> x.slot < slot) (state ^. (_currentMarloweState <<< _executionState))
```

Comment:
oh, this looks like the function that I added all the comments to above, perhaps it should all be here?

Reply:
Sorry, I am not sure how we can extract the slot as a Maybe using lenses

Reply:
`preview (_currentMarloweState <<< _executionState <<< _Just <<< _slot) state`